release/.server.config

```diff
@@ -0,0 +1,17 @@
+#
+# Template configuration file for server environment
+# used by the release shell scripts.
+#
+
+# Base PATH where VCS checkouts are located
+# Code is located in version specific subdirectories named 'squid-*'
+SQUID_VCS_PATH="/var/local/vcs"
+
+# Base PATH where WWW server files are located
+SQUID_WWW_PATH="/var/www"
+
+# Base PATH where rsync shares are located
+SQUID_RSYNC_PATH="/var/rsync"
+
+# Squid build server
+SQUID_CI_BUILD="example.com.invalid"
```

Comment:
CI is too general IMO -- we have at least three CI servers. I do not know what "build server" is so I cannot suggest specific changes, but please _detail_ that in the comment/description and then rename the variable (e.g., to `SQUID_BUILD_SERVER` or something more suitable).